lib/Zonemaster/Backend/DB.pm

```diff
@@ -130,6 +133,79 @@ sub process_unfinished_tests {
     }
 }
 
+sub _normalize_parameter_hash {
+    my ( $self, $params ) = @_;
+
+    my $profile = Zonemaster::Engine::Profile->effective;
+
+    my %normalized = ();
+    $normalized{domain}   = lc $$params{domain} // "";
+    $normalized{ipv4}     = $$params{ipv4}      // $profile->get( 'net.ipv4' );
+    $normalized{ipv6}     = $$params{ipv6}      // $profile->get( 'net.ipv6' );
+    $normalized{profile}  = $$params{profile}   // "default";
+    $normalized{priority} = $$params{priority}  // 10;
+    $normalized{queue}    = $$params{queue}     // 0;
```

Comment:
Both "priority" and "queue" are set in RPCAPI.pm. They should not be set twice.

Reply:
Well, to be honest there is also a duplication in each SQL schema as those values are the default for their respective column.
I agree that there are too many duplicates of these values. However, I need them in this new method for the unit tests.
I haven't removed the initialization in the RPCAPI.pm file because I feel that even though they have the same value, they don't exactly have the same meaning (it is an initialization instead of a fall back to some default values).

I started some refactoring of all these values duplication.

Reply:
It could be OK to have these values here if it otherwise create other problems, but then it should be documented in the code that the values are also set in RPCAPI.pm. In a second step (later PR) it could be cleaned up.

Reply:
Is it documented somewhere that priority and queue get default values in two places of the code?

Reply:
Oops, I forgot this one. I'll do it.